keyring.go

```diff
@@ -17,6 +17,38 @@ type PublicKeyRequest struct {
 	KeyID KeyID
 }
 
+// PublicKeyNotExpired is a magic value for PublicKeyLookupResult.ExpiredTS:
+// it indicates that this is an active key which has not yet expired
+const PublicKeyNotExpired = Timestamp(0)
+
+// PublicKeyNotValid is a magic value for PublicKeyLookupResult.ValidUntilTS:
+// it is used when we don't have a validity period for this key. Most likely
+// it is an old key with an expiry date.
+const PublicKeyNotValid = Timestamp(0)
```

Comment:
Are these two meant to have the same values?

Reply:
it's fine that they are the same value: they end up in different fields.

Reply:
Fair